### ISSUE TITLE
Installation Documentation

### ISSUE THREAD

**Issue by terrytibbles** [Icinga/icingaweb2#2979]
Hello,

I'm installing the latest version of Icinga Web 2 and the installation instructions stop at [starting the web setup](https://www.icinga.com/docs/icingaweb2/latest/doc/02-Installation/#starting-web-setup). After this, the [database IDO modules need to be installed](https://www.icinga.com/docs/icinga2/latest/doc/02-getting-started/#installing-the-ido-modules-for-mysql) and the [IDO schema needs to be imported](https://www.icinga.com/docs/icinga2/latest/doc/02-getting-started/#setting-up-the-mysql-database). Also, am I right in thinking that Icinga Web 2 uses the 'icinga2' database for IDO, and not 'icinga', listed in the documentation (see latest link)?

Thanks.

**Comment by dnsmichi** [comment]
The Debian packages use a different default which is noted in the Icinga 2 docs - `icinga2`. The main default on RPM based systems is `icinga`.

The IDO requirement is listed above, so you'll have to have Icinga 2 with DB IDO fully running and enabled. Though it might not be clear enough what exactly happens inside the setup wizard.

I'll think about slight adoptions once #2972 is merged.

**Comment by terrytibbles** [comment]
[This](https://www.icinga.com/docs/icinga2/latest/doc/02-getting-started/#installing-the-ido-modules-for-mysql) section I find very confusing. There is a note at the bottom of the section, which applies to Debian/Ubuntu users, and says:

> The Debian/Ubuntu packages provide a database configuration wizard by default. You can skip the automated setup and install/upgrade the database manually if you prefer that.

It's not clear if Debian/Ubuntu users should use the wizard, and if they do, whether they still need to 
[set up the MySQL database](https://www.icinga.com/docs/icinga2/latest/doc/02-getting-started/#setting-up-the-mysql-database). Not mentioning the difference between the database names on the different platforms makes this very confusing.

**Comment by dnsmichi** [comment]
That's unrelated to the Icinga Web 2 docs though. Please move that into a separate ticket for Icinga 2.

**Comment by terrytibbles** [comment]
I had to follow those instructions to set up Icinga Web 2.

**Comment by dnsmichi** [comment]
As said, if there's anything wrong with the Icinga 2 documentation, please open an issue over there. The Icinga Web 2 setup wizard shouldn't be documented for the moment, as it should be self explanatory which values to choose.
